lib/mindwendel_web/live/live_helpers.ex

```diff
@@ -12,6 +12,14 @@ defmodule MindwendelWeb.LiveHelpers do
     Enum.member?(brainstorming.moderating_users |> Enum.map(& &1.id), current_user.id)
   end
 
+  def has_ownership(idea, current_user) do
+    idea.user_id == current_user.id
```

Comment:
I'd say that's more of a business domain thing and should hence live in a context like the `ideas` context or similar and then be called from the view (wrapped in a helper or not)